network/framework/src/application/storage.rs

```diff
@@ -23,6 +24,11 @@ use std::{
     ops::Deref,
     sync::{Arc, RwLockWriteGuard},
 };
+use tokio::sync::mpsc::error::TrySendError;
+
+// notification_backlog is how many ConnectionNotification items can be queued waiting for an app to receive them.
+// Beyond this, new messages will be dropped if the app is not handling them fast enough.
+const NOTIFICATION_BACKLOG: usize = 100;
```

Comment:
nit: maybe make this `200`+ so we get some buffer? e.g.,, we're at ~120 connected peers in `mainnet`, so maybe under some weird circumstances it's possible that the entire 100 could be filled at once? Should be unlikely, but seems safer to over-approximate.

Reply:
Gonna make it 1000 so that it should be big enough for any initial surge of notifying _all_ the connected peers, given the `.subscribe()` change where it pre-queues all the peers as new connection notifications.